src/ServiceControlInstaller.PowerShell/ServiceControlInstaller.Powershell.dll-help.xml

```diff
@@ -1395,134 +1631,158 @@ Get-SecurityIdentifier MyDomain\Foo</dev:code>
 		<dev:version />
 	</command:details>
 	<maml:description>
-		<maml:para>Create an instance from an unattended install file.  The resultant XML file can be passed to the ServiceControl Installation executable using the UNATTENDEDFILE command line option.  An instance can also me made from the file by using the New-ServiceControlInstanceFromUnattendedFile cmdlet.</maml:para>
+		<maml:para>Create an instance from an unattended install file.  The resultant XML file can be passed to the ServiceControl Installation executable using the UNATTENDEDFILE command line option.  An instance can also me made from the file by using the New-ServiceControlInstanceFromUnattendedFile cmdlet.
+</maml:para>
 	</maml:description>
 	<command:syntax>
 		<command:syntaxItem>
 			<maml:name>New-ServiceControlUnattendedFile</maml:name>
 			<command:parameter required="true" variableLength="false" globbing="false" pipelineInput="false" position="named">
 				<maml:name>Name</maml:name>
 				<maml:description>
-					<maml:para />
+					<maml:para>The name of the ServiceControl Instance.  This will be used as the Windows Service Name.  The name must be unique for all Windows services and not contain any spaces or special characters.</maml:para>
```

Comment:
I feel like there should be a word between `...Windows services and ` and the `not contain any ...`

> ...Windows services and can not container any spaces...